systemtests/src/main/java/io/enmasse/systemtest/selenium/page/ConsoleWebPage.java

```diff
@@ -584,8 +585,16 @@ public void createAddressWebConsole(Destination destination, boolean openConsole
         //select address type
         selenium.clickOnItem(selenium.getWebElement(() -> selenium.getDriver().findElement(By.id(destination.getType()))), "Radio button " + destination.getType());
 
-        WebElement nextButton = selenium.getWebElement(() -> selenium.getDriver().findElement(By.id("nextButton")));
+        //if address type is subscription, fill in the topic dropdown box
+        if (destination.getType().equals(AddressType.SUBSCRIPTION.toString())) {
+            log.info("Selecting topic to attach subscription to");
+            WebElement topicDropDown = selenium.getWebElement(() -> selenium.getDriver().findElement(By.className("combobox")));
+            selenium.clickOnItem(topicDropDown);
+            Select combobox = new Select(topicDropDown);
+            combobox.selectByVisibleText(destination.getAddress().replace("-subscriber", ""));
```

Comment:
Rather than extracting the name of the topic from the subscribers address, I would add a getTopic() method to Destination and then pass in a Destination with the topic set explicitly.